src/schemas/execute.yaml

```
@@ -168,10 +168,7 @@ Withdrawal:
       $ref: '#/components/schemas/uint64'
 EthSimulateResult:
   title: Full results of eth_simulate
-  type: array
-  oneOf:
-    - $ref: '#/components/schemas/EthSimulateBlockResultInvalid'
-    - $ref: '#/components/schemas/EthSimulateBlockResultSuccess'
+  $ref: '#/components/schemas/EthSimulateBlockResultSuccess'
 EthSimulateBlockResultSuccess:
```

Can probably just delete `EthSimulateResult` and rename `EthSimulateResultSuccess` to `EthSimulateResult` now?
```suggestion
EthSimulateResult:
```